transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
```

Comment:
Maybe nothing and remove the pylint restriction? The pylint config is not the law in my opinion, but a safeguard. If it means adding useless comments, I'd rather remove the restriction.